lerobot/configs/default.yaml

```diff
@@ -57,3 +57,29 @@ wandb:
   disable_artifact: false
   project: lerobot
   notes: ""
+
```

Comment:
We don't have full clarity on what goes in `default.yaml` but in my mind it's kind of thinks that are common to multiple types of config files (policy, env), or things that don't belong in either like the run dir and job name.

But I do think this section goes under policy's `training` section maybe? These transforms are training-time only. They are part of the training recipe.

Reply:
I agree with you, but I also think it'd be quite nested this way. I don't have strong opinions about this. cc @Cadene

Reply:
Let's have them here ;)

Reply:
Fixed https://github.com/huggingface/lerobot/pull/234/commits/7a097e9e9f78b92f54ab2b53c214b16a9c58310d